libmscore/stringdata.cpp

```diff
@@ -47,6 +47,14 @@ StringData::StringData(int numFrets, QList<instrString>& strings)
             stringTable.append(i);
       }
 
+// called from import (musicxml/guitarpro/...)
+void StringData::set(const StringData& src)
+      {
+          *this = src;
+          if (stringTable.size() == 5)
```

Comment:
What about other plucked string instruments with 5 strings, like some basses?

Reply:
configBanjo5thString() does the needed checks to assure it is a 5 string banjo